sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/README.md

```diff
@@ -1,34 +1,54 @@
 # Azure Monitor Exporter client library for .NET
 
-TODO
+The [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet) exporters which send [telemetry data](https://docs.microsoft.com/en-us/azure/azure-monitor/app/data-model) to the [Azure Monitor Platform](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) following the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
 
 ## Getting started
 
-### Install the package
+### Prerequisites
 
+- **Azure Subscription:**  To use Azure services, including Azure Monitor Exporter for [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet), you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+- **Azure Application Insights Connection String:** To send telemetry data to the monitoring service you'll need connection string from Azure Application Insights. If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [Create an Application Insights resource](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource) and [copy the connection string](https://docs.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=net#finding-my-connection-string).
 
-### Prerequisites
+### Install the package
 
+Install the Azure Monitor Exporter for OpenTelemetry .NET with NuGet:
+
+```shell
+dotnet add package OpenTelemetry.Exporter.AzureMonitor
+```
 
 ### Authenticate the client
 
+For the exporter to send data to Azure Monitor use a connection string, which is created automatically when creating an Application Insights resource.
 
 ## Key concepts
 
-TODO
+This exporter sends traces to the configured Azure Monitor Resource using HTTP. IP address used by the Azure Monitor is documented in [IP addresses used by Application Insights and Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/app/ip-addresses#outgoing-ports).
 
 ## Examples
 
-TODO
+Refer to [`DemoTrace.cs`](tests/OpenTelemetry.Exporter.AzureMonitor.Demo.Tracing/DemoTrace.cs) for a complete demo.
+
+```csharp
+OpenTelemetry.Sdk.CreateTracerProviderBuilder()
+    .SetResource(resource)
+    .AddSource("Samples.SampleClient")
+    .AddAzureMonitorTraceExporter(o => {
+        o.ConnectionString = $"InstrumentationKey=Ikey;";
```

Comment:
if we were asking to copy connectingstring, then they should be able to paste it as is here, right?

Reply:
Yes. You're correct.

Reply:
You can use this: `InstrumentationKey=00000000-0000-0000-0000-000000000000`. This is the minimal valid connection string, and this is the one I've used in all the documentation.

Reply:
there is no mention of instrumentationkey ever in this doc, so the example should also avoid mentioning it.

`o.ConnectionString = "<actualconnectionstring>"`